src/modules/link-prefetch/constants.ts

```diff
@@ -0,0 +1 @@
+export const cacheName: string = 'AMP-PREFETCHED-LINKS';
```

Comment:
Let's capitalize this constant, for consistency.

Also, you might consider calling it something more explicit since it's used outside of the `link-prefetch` component now.